app/views/reports/regions/details.html.erb

```diff
@@ -1,208 +1,206 @@
 <%= render "header", period_selector: false %>
-<% if @region.is_a?(Facility) %>
-  <% if current_admin.accessible_facilities(:view_reports).include?(@region) %>
```

Comment:
We are already authorizing the region in the [controller](https://github.com/simpledotorg/simple-server/blob/master/app/controllers/reports/regions_controller.rb#L156-L182), so we don't need to do it again here.